app/src/main/java/com/berniesanders/connect/MainActivity.java

```diff
@@ -32,8 +48,21 @@ protected void onCreate(Bundle savedInstanceState) {
         mText.setText(mContext.getPackageName());
 
         mActionAlertsModel.getActionAlerts().subscribe(
```

Comment:
I'll make it an object, since we will want this accessible anywhere where there will be a network request.